zip-0227.rst

```diff
@@ -205,15 +210,18 @@ The relations between the Asset Identifier, Asset Digest, and Asset Base are sho
     :align: center
     :figclass: align-center
 
-    Diagram relating the Asset Identifier, Asset Digest, and Asset Base in the ZSA Protocol
+    Diagram relating the Asset Identifier, Asset Digest, and Asset Base in the Orchard-ZSA Protocol
 
 
-**Note:** To keep notations light and concise, we may omit :math:`\mathsf{AssetId}` (resp. :math:`\mathsf{Protocol}`) in the subscript (resp. superscript) when the Asset Identifier (resp. Protocol) is clear from the context.
+**Note:** To keep notations light and concise, we may omit :math:`\mathsf{AssetId}` in the subscript when the Asset Identifier is clear from the context.
 
-Wallets MUST NOT display just the :math:`\mathsf{asset\_desc}` string to their users as the name of the Asset. Some possible alternatives include:
+The Asset Base is the unique identifier of the Asset in the Orchard-ZSA Protocol.
+Wallets MUST NOT display just the :math:`\mathsf{asset\_desc}` string to their users as the name of the Asset.
+If the :math:`\mathsf{asset\_desc}` string is not a well-formed UTF-8 code unit sequence, the wallet MUST provide a warning that the Asset description cannot be parsed as a UTF-8 encoded string, and SHOULD use the byte string as is in their method for displaying the Asset description.
+
+Wallets could allow clients to provide an additional configuration file that stores a one-to-one mapping of names to Asset Bases via a petname system. This allows clients to rename the Assets in a way they find useful. Default versions of this file with well-known Assets listed can be made available online as a starting point for clients.
```

Comment:
avoid the term `petname system`. use auxiliary naming system.

Reply:
Done